package.json

```diff
@@ -14,32 +14,65 @@
     "url": "https://github.com/woocommerce/woocommerce/issues"
   },
   "scripts": {
+    "build-storybook": "build-storybook  -c ./tools/storybook/.storybook",
     "preinstall": "npx only-allow pnpm",
     "postinstall": "pnpm git:update-hooks",
-    "git:update-hooks": "rm -r .git/hooks && mkdir -p .git/hooks && husky install"
+    "git:update-hooks": "rm -r .git/hooks && mkdir -p .git/hooks && husky install",
+    "storybook": "./tools/storybook/import-wp-css-storybook.sh && BABEL_ENV=storybook STORYBOOK=true start-storybook -c ./tools/storybook/.storybook -p 6007 --ci",
+		"storybook-rtl": "USE_RTL_STYLE=true pnpm run storybook"
   },
   "devDependencies": {
     "@automattic/nx-composer": "^0.1.0",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/runtime": "^7.17.2",
     "@nrwl/cli": "^13.3.4",
     "@nrwl/devkit": "^13.1.4",
     "@nrwl/linter": "^13.3.4",
     "@nrwl/tao": "13.3.4",
     "@nrwl/web": "^13.3.4",
     "@nrwl/workspace": "^13.3.4",
+    "@storybook/addon-a11y": "^6.4.19",
+		"@storybook/addon-actions": "^6.4.19",
+		"@storybook/addon-console": "^1.2.3",
+		"@storybook/addon-controls": "^6.4.19",
+		"@storybook/addon-docs": "^6.4.19",
+		"@storybook/addon-knobs": "^6.4.0",
+		"@storybook/addon-links": "^6.4.19",
+		"@storybook/addon-storysource": "^6.4.19",
+		"@storybook/addon-viewport": "^6.4.19",
+		"@storybook/addons": "^6.4.19",
+		"@storybook/api": "^6.4.19",
+		"@storybook/builder-webpack5": "^6.4.19",
+		"@storybook/components": "^6.4.19",
+		"@storybook/core-events": "^6.4.19",
+		"@storybook/manager-webpack5": "^6.4.19",
+		"@storybook/react": "^6.4.19",
+		"@storybook/theming": "^6.4.19",
     "@types/node": "14.14.33",
     "@woocommerce/eslint-plugin": "workspace:*",
     "@wordpress/eslint-plugin": "^11.0.0",
     "@wordpress/prettier-config": "^1.1.1",
+    "@woocommerce/data": "workspace:*",
```

Comment:
If this is for webpack config, should this be `"@wordpress/data": "^6.3.0",?

Reply:
Nice catch! Fixed in https://github.com/woocommerce/woocommerce/pull/32729/commits/889a7fdeef9b0335b46b9b1a76a6d93e89926fa1